### PR TITLE
fix: navigation e2es

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Used the e2e files from `v2.0-tmp`

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Review by rikukissa — APPROVED** [review]

**Review by cibelius — APPROVED** [review]

File: e2e/testcases/navigation/navigating-in-and-out-of-action.spec.ts [hunk 1]
```diff
@@ -86,6 +86,6 @@ test.describe.serial('Navigating in and out of action', () => {
 
   test('Browser forward button should take user back to the event overview page', async () => {
```
Comment (cibelius) [inline e2e/testcases/navigation/navigating-in-and-out-of-action.spec.ts:87]: is this text correct anymore?
